src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import usersFromServer from '../../api/users';
+import { getUserById } from '../../service/user';
+
+type Props = {
+  onSubmit: (todo: Todo) => void;
+};
+
+export const NewTodo: React.FC<Props> = ({ onSubmit }) => {
+  const [userId, setUserId] = useState(0);
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setTitleError] = useState(false);
+  const [hasSelectionError, setSelectionError] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setTitleError(false);
+  };
+
+  const handleUserIdChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+    setSelectionError(false);
+  };
+
+  const reset = () => {
+    setTitle('');
+    setUserId(0);
+  };
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    setTitleError(!title);
+    setSelectionError(!userId);
+
+    if (!title || !userId) {
```

Comment:
i can create the todo only with spaces
```suggestion
    if (!title.trim() || !userId) {
```